futurenhs.app/services/postGroupMemberInvite/index.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    setFetchOpts as setFetchOptionsHelper,
+    fetchJSON as fetchJSONHelper,
+} from '@helpers/fetch'
+import { services } from '@constants/services'
+import { requestMethods, defaultTimeOutMillis } from '@constants/fetch'
+import { ServiceError } from '..'
+import { ServerSideFormData } from '@helpers/util/form'
+import { ServiceResponse } from '@appTypes/service'
+import { User } from '@appTypes/user'
+
+declare type Options = {
+    user: User
+    groupId: string
+    headers?: any
+    body: FormData | ServerSideFormData
+}
+
+declare type Dependencies = {
+    setFetchOptions: any
+    fetchJSON: any
+}
+
+export const postGroupMemberInvite = async (
+    { user, headers = {}, body, groupId }: Options,
+    dependencies?: Dependencies
+): Promise<ServiceResponse<null>> => {
+    const setFetchOptions =
+        dependencies?.setFetchOptions ?? setFetchOptionsHelper
+    const fetchJSON = dependencies?.fetchJSON ?? fetchJSONHelper
+
+    const { id } = user
+    const emailAddress: FormDataEntryValue = body.get('Email')
+    /**
+     * TODO: Actual endpoint TBD
+     */
+    const apiUrl: string = `${process.env.NEXT_PUBLIC_API_GATEWAY_BASE_URL}/v1/users/${id}/groups/${groupId}/invite`
+    const apiResponse: any = await fetchJSON(
+        apiUrl,
+        setFetchOptions({
+            method: requestMethods.POST,
+            headers: headers,
+            body: {
+                emailAddress: emailAddress,
+            },
+        }),
+        defaultTimeOutMillis
+    )
+
+    const apiMeta: any = apiResponse.meta
+    const apiData: any = apiResponse.json
+
+    const { ok, status, statusText } = apiMeta
+
+    if (!ok) {
+        throw new ServiceError(
+            'An unexpected error occurred when attempting to invite a group member',
+            {
+                serviceId: services.POST_SITE_USER_INVITE,
```

Comment:
Needs a unique service id

Reply:
Cheers! Already set up a new one just hadn't referenced it, resolved.